build.gradle

```diff
@@ -202,7 +202,13 @@ project.ext.externalDependency = [
     'testContainersCassandra': 'org.testcontainers:cassandra:' + testContainersVersion,
     'typesafeConfig':'com.typesafe:config:1.4.1',
     'wiremock':'com.github.tomakehurst:wiremock:2.10.0',
-    'zookeeper': 'org.apache.zookeeper:zookeeper:3.4.14'
+    'zookeeper': 'org.apache.zookeeper:zookeeper:3.4.14',
+    'wire': 'com.squareup.wire:wire-compiler:3.7.1',
+    'charle':  'com.charleskorn.kaml:kaml:0.53.0',
+    'jsonv': 'org.json:json:20230227',
```

Comment:
Looks like this is duplicated above `'json': 'org.json:json:20230227',` is the same as this one. Please consolidate to one.